### PR TITLE
Remove redux

### PR DESCRIPTION
**Pull request by Nepomuceno**
### What does this PR do?

Removing dependency on redux moving all to depend upon react query.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?

### CODE REVIEW

File: client/src/api/utils/AzureBlob.ts [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
   return dataSource;
 }
 
-export function getContainerClient(account: string, container: string): ContainerClient {
-  const dataSource = getDataSourceFromConnection(account, container);
+export function getContainerClient(dataSources: Record<string,DataSource>, account: string, container: string): ContainerClient {
+  const dataSource = getDataSourceFromDatasources(dataSources, account, container);
   if (!dataSource || !dataSource.sasToken) {
+    
+   console.log(`DATASOURECE NOT FOUND: ${account}/${container}`, dataSources);
```
Comment (PaulDFoster) [inline client/src/api/utils/AzureBlob.ts]: Spelling nit: DATASOURCE

File: client/src/api/utils/AzureBlob.ts [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
     throw new Error('No connection found');
   }
+  console.log(`DATASOURECE FOUND: ${dataSource.account}/${dataSource.container} ${dataSource.sasToken}`);
```
Comment (PaulDFoster) [inline client/src/api/utils/AzureBlob.ts:30]: ditto

**Review by PaulDFoster — COMMENTED** [review]
A spelling error duplicated in two places.

**Review by PaulDFoster — APPROVED** [review]

**Comment by 777arc** [comment]
looks like the window resizing in the spectrogram page got screwed up, ill see if i can fix

**Review by 777arc — APPROVED** [review]

**Pull request merged**